include/kernels/dslash_gamma_helper.cuh

```diff
@@ -84,6 +84,16 @@ namespace quda {
       case 3: arg.out(x_cb, parity) = in.gamma(3); break;
       case 4: arg.out(x_cb, parity) = in.gamma(4); break;
       }
+      if (arg.doublet) {
+        ColorSpinor<typename Arg::real, Arg::nColor, 4> in = arg.in(x_cb + 1 * arg.volumeCB, parity);
```

Comment:
We can remove the `1 *`, though I imagine any compiler would optimize this out.

Reply:
The compiler does indeed optimize this out.  I have done a bit of cleanup to this function to make it a bit cleaner though e7a54b3c4069b2674d6d423eccfe09ea1d40078d.